tests/behaviour/cube.feature

```diff
@@ -257,4 +257,68 @@ Feature: Cube!
           {'csv_column_title': 'Measure', 'structural_definition': {'measures': [{'label': 'Length', 'description': None, 'parent_measure_uri': None, 'source_uri': None, 'uri_safe_identifier_override': None, 'arbitrary_rdf': []}]}, 'csv_column_uri_template': None, 'uri_safe_identifier_override': None}, 
           {'csv_column_title': 'Unit', 'structural_definition': {'units': [{'label': 'cm', 'description': None, 'source_uri': None, 'uri_safe_identifier_override': None, 'arbitrary_rdf': [], 'base_unit': None, 'base_unit_scaling_factor': None, 'qudt_quantity_kind_uri': None, 'si_base_unit_conversion_multiplier': None}]}, 'csv_column_uri_template': None, 'uri_safe_identifier_override': None}
       ]
+    """
+
+    Scenario: Output a cube when the code list is defined using code list config schema v1.0 and when there are references to concepts defined elsewhere.
+    Given The config json file "v1.1/cube_config_with_concepts_defined_elsewhere.json" and the existing tidy data csv file "v1.1/data.csv"
+    When The cube is created
+    Then The cube Metadata should match
+      """
+      {
+        "title": "Test cube config",
+        "identifier": None,
+        "summary": "Test cube config summary",
+        "description": "Test cube config description",
+        "creator_uri": "http://statistics.data.gov.uk",
+        "publisher_uri": "http://statistics.data.gov.uk",
+        "landing_page_uris": [],
+        "theme_uris": [],
+        "keywords": ["Test cube"],
+        "dataset_issued": "2022-04-08",
+        "dataset_modified": None,
+        "license_uri": "https://creativecommons.org/licenses/by/4.0/",
+        "public_contact_point_uri": None,
+        "uri_safe_identifier_override": None
+      }
+      """
+    Then The cube Columns should match
+    """
```

Comment:
Can we avoid reusing these kinds of tests where we match on JSON representations of objects? They're really brittle and it's hard to figure out what you're actually testing.

Can't you test something clear like the code-list's CSV output has the appropriate column defined?

Reply:
@robons I have been following the same style used in the previous tests in this .feature file. But you have a point when it comes to their robustness and readability. I will update accordingly.

Reply:
Yeah, I want to get rid of those other ones too.

Reply:
@robons The behave tests in cube feature now assert the cols and data instead of matching JSON representations of objects. I have updated all the tests within the cube feature file.